.coveragerc

```diff
@@ -1,8 +1,14 @@
 [run]
 branch = True
 source = dallinger
+parallel = True
 
 [report]
-fail_under = 41
+fail_under = 55
 show_missing = True
 skip_covered = True
+
+[paths]
+source =
+    dallinger
+    .tox/py27/lib/python2.7/site-packages/dallinger
```

Comment:
More or less yeah, it treats these two locations as equivalent when merging coverage output.